README.md

```diff
@@ -46,168 +49,171 @@ With their functions being:
+`insider`       |insider trading of the company |[Finviz](https://finviz.com/)
+`sec`           |SEC filings |[MarketWatch](https://www.marketwatch.com/)
+`short`         |short interest |[Quandl](https://www.quandl.com/)
+`warnings`      |company warnings according to Sean Seah book |[MarketWatch](https://www.marketwatch.com/)
+
 
 ### [Prediction Techniques](prediction_techniques/README.md) <a name="Prediction_Techniques"></a>
-```
-sma         simple moving average
-knn         k-Nearest Neighbors
-linear      linear regression (polynomial 1)
-quadratic   quadratic regression (polynomial 2)
-cubic       cubic regression (polynomial 3)
-regression  regression (other polynomial)
-arima       autoregressive integrated moving average
-prophet     Facebook's prophet prediction
-mlp         MultiLayer Perceptron
-rnn         Recurrent Neural Network
-lstm        Long-Short Term Memory
-```
+Command|Technique|Wikipedia|Investopedia
+------ | ------------|---|----
+`sma`         |simple moving average | [Link](https://en.wikipedia.org/wiki/Moving_average#Simple_moving_average)| [Link](https://www.investopedia.com/terms/s/sma.asp)
+`knn`         |k-Nearest Neighbors | [Link](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm)
+`linear`      |linear regression (polynomial 1) | [Link](https://en.wikipedia.org/wiki/Linear_regression)| [Link](https://www.investopedia.com/terms/r/regression.asp)
+`quadratic`   |quadratic regression (polynomial 2) | [Link](https://en.wikipedia.org/wiki/Polynomial_regression)| [Link](https://www.investopedia.com/terms/r/regression.asp)
+`cubic`       |cubic regression (polynomial 3) | [Link](https://en.wikipedia.org/wiki/Polynomial_regression)| [Link](https://www.investopedia.com/terms/r/regression.asp)
+`regression`  |regression (other polynomial) | [Link](https://en.wikipedia.org/wiki/Polynomial_regression)| [Link](https://www.investopedia.com/terms/r/regression.asp)
+`arima`       |autoregressive integrated moving average | [Link](https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average)| [Link](https://www.investopedia.com/terms/a/autoregressive-integrated-moving-average-arima.asp) 
+`prophet`     |Facebook's prophet prediction ([Link](https://facebook.github.io/prophet/))|
+`mlp`         |MultiLayer Perceptron | [Link](https://en.wikipedia.org/wiki/Multilayer_perceptron)
+`rnn`         |Recurrent Neural Network  | [Link](https://en.wikipedia.org/wiki/Recurrent_neural_network)
+`lstm`        |Long Short-Term Memory  | [Link](https://en.wikipedia.org/wiki/Long_short-term_memory)
+
 
 ## Install <a name="Install"></a>
 
 This project was written and tested with Python 3.6.8.
 
-In order to install all libraries used by this repository, you must run
+In order to install, you must run:
 ```
-pip install -r requirements.txt
+git clone https://github.com/DidierRLopes/GamestonkTerminal
+cd GamestonkTerminal/
+pip3 install -r requirements.txt
 ```
+Then run:
+```
+python3 gamestonk_terminal.py
```

Comment:
We're working on fixing the installation instructions, no need to change this for now :)

Also, the pip3 and python3 is due to the fact that you must have python 2.x installed in your machine.

When Python 3.x is installed and you add his path to the Environment variables, it will see that you already have a python variable pointing to a python 2 executable. Hence, it will use python3 to point at python 3 executable.

A way you can get around this easily is just overriding the alias with:
```
alias python=python3
```